apps/settings/elements/messaging.html

```diff
@@ -23,8 +23,14 @@ <h2 data-l10n-id="message-confirmation"></h2>
       <ul class="mmsSettings-list">
         <li id="menuItem-readReport" aria-disabled="true">
```

Comment:
@julienw  will change the id at last, this will led to change at other place too. SO, will do at last.

menuItem-readReport